sdk/management/azure/src/main/java/com/azure/management/Azure.java

```diff
@@ -405,575 +405,474 @@ public Azure withDefaultSubscription() throws CloudException, IOException {
-     * @return entry point to managing network interfaces
-     */
+    /** @return entry point to managing network interfaces */
     public NetworkInterfaces networkInterfaces() {
         return this.networkManager.networkInterfaces();
     }
 
-    /**
-     * @return entry point to managing compute resource usages
-     */
+    /** @return entry point to managing compute resource usages */
     public ComputeUsages computeUsages() {
         return computeManager.usages();
     }
 
-    /**
-     * @return entry point to managing key vaults
-     */
+    /** @return entry point to managing key vaults */
     public Vaults vaults() {
         return this.keyVaultManager.vaults();
     }
 
-//    /**
-//     * @return entry point to managing batch accounts.
-//     */
-//    public BatchAccounts batchAccounts() {
-//        return batchManager.batchAccounts();
-//    }
-
-//    /**
-//     * @return entry point to managing traffic manager profiles.
-//     */
-//    public TrafficManagerProfiles trafficManagerProfiles() {
-//        return trafficManager.profiles();
-//    }
-//
-//    /**
-//     * @return entry point to managing Redis Caches.
-//     */
-//    public RedisCaches redisCaches() {
-//        return redisManager.redisCaches();
-//    }
-//
-//    /**
-//     * @return entry point to managing cdn manager profiles.
-//     */
-//    public CdnProfiles cdnProfiles() {
-//        return cdnManager.profiles();
-//    }
-
-//    /**
-//     * @return entry point to managing DNS zones.
-//     */
-//    public DnsZones dnsZones() {
-//        return dnsZoneManager.zones();
-//    }
-
-    /**
-     * @return entry point to managing web apps.
-     */
+    //    /**
+    //     * @return entry point to managing batch accounts.
+    //     */
+    //    public BatchAccounts batchAccounts() {
+    //        return batchManager.batchAccounts();
+    //    }
+
+    //    /**
+    //     * @return entry point to managing traffic manager profiles.
+    //     */
+    //    public TrafficManagerProfiles trafficManagerProfiles() {
+    //        return trafficManager.profiles();
+    //    }
+    //
+    //    /**
+    //     * @return entry point to managing Redis Caches.
+    //     */
+    //    public RedisCaches redisCaches() {
+    //        return redisManager.redisCaches();
+    //    }
+    //
+    //    /**
+    //     * @return entry point to managing cdn manager profiles.
+    //     */
+    //    public CdnProfiles cdnProfiles() {
+    //        return cdnManager.profiles();
+    //    }
+
+    //    /**
+    //     * @return entry point to managing DNS zones.
+    //     */
+    //    public DnsZones dnsZones() {
+    //        return dnsZoneManager.zones();
+    //    }
+
+    /** @return entry point to managing web apps. */
     public WebApps webApps() {
         return appServiceManager.webApps();
     }
 
-    /**
-     * @return entry point to managing app services.
-     */
-    public AppServiceManager appServices() {
```

Comment:
Class in implementation is not allowed to be exposed. So use common approach of exposing all its sub interface instead.